packages/mobile/src/screens/collection-screen/CollectionScreenDetailsTile.tsx

```diff
@@ -153,25 +182,42 @@ const recordPlay = (id: Maybe<number>, play = true) => {
 
 export const CollectionScreenDetailsTile = ({
   description,
-  extraDetails = [],
   collectionId,
   isAlbum,
-  isPrivate,
   isPublishing,
   renderImage,
   trackCount: trackCountProp,
-  isOwner,
+  isOwner = false,
   hideOverflow,
   hideRepost,
   hideFavorite,
   hasStreamAccess,
   streamConditions,
   ddexApp,
-  isDeleted,
-  ...detailsTileProps
+  hideShare,
```

Comment:
this is a great start, ideally we just drop the "detailstile" nomeclature and move this all to the collection-screen